src/foam/u2/wizard/AlternateFlow.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.wizard',
+  name: 'AlternateFlow',
+
+  imports: [
+    'wizardlets?'
+  ],
+
+  properties: [
+    // IDEA: maybe these two properties can come form a mixin
+    {
+      class: 'String',
+      name: 'name'
+    },
+    {
+      class: 'String',
+      name: 'label'
+    },
+
```

Comment:
Why this blank line?

Reply:
I found it easier to read like this, because it's easier to associate the above comment with the properties it relates to